### PR TITLE
Fix letter: positioning of osdviewer and incoming links 

### LINKED ISSUE #125: Letter: positioning of osdviewer and incoming links
There is a problem with the margin between the osdviewer and the incoming links section. This works in the resource template but not in the letter template. In the letter template, `<section>` is used.

screen shot 2019-03-06 at 2 48 55 pm

### PR DESCRIPTION
**Pull request by flavens**
closes #125

### CODE REVIEW

**Comment by flavens** [comment]
[Fix letter css](https://trello.com/c/HJc5fwy4/61-fix-letter-css)

File: src/app/resource/letter/letter.component.html [hunk 1]
```diff
@@ -145,27 +148,23 @@ <h4 class="subheading-2">{{ontologyInfo?.getLabelForProperty(props?.sysnum[0].pr
...
+                        <span class="link" (click)="showIncomingRes(incomingLinkSourceRes.id, incomingLinkSourceRes.type)">{{incomingLinkSourceRes.label}}</span>
+                    </mat-list-item>
+                </mat-list>
+            </div>
 
         </div>
     </div>
 
-</div>
+</div>
```
Comment (tobiasschweizer) [inline src/app/resource/letter/letter.component.html]: Could you add a newline at the end of the file?

File: src/app/resource/letter/letter.component.scss [hunk 2]
```diff
@@ -11,3 +11,7 @@ h4.subheading-2 {
   height: auto !important;
   margin: 6px 0;
 }
+
+.incoming-link-position-letter {
+  margin-top: 110px;
+}
```
Comment (tobiasschweizer) [inline src/app/resource/letter/letter.component.scss]: Could you add a newline at the end of the file?

**Review by tobiasschweizer — CHANGES_REQUESTED** [review]

**Comment by tobiasschweizer** [comment]
@flavens The CSS bug is fixed, thanks a lot!

**Review by tobiasschweizer — APPROVED** [review]

**Pull request merged**